RecoEgamma/EgammaPhotonProducers/src/ConversionGSCrysFixer.cc

```diff
@@ -0,0 +1,72 @@
+#include <memory>
+/** \class ConversionGSCrysFixer
+ **
+ **
+ **  \author Yutaro Iiyama, MIT
+ **
+ ***/
+
+#include "RecoEgamma/EgammaPhotonProducers/interface/ConversionGSCrysFixer.h"
+
+#include "DataFormats/EgammaCandidates/interface/Conversion.h"
+#include "DataFormats/EgammaReco/interface/SuperClusterFwd.h"
+#include "DataFormats/EcalDetId/interface/EcalSubdetector.h"
+#include "DataFormats/Common/interface/RefToPtr.h"
+
+#include "RecoEgamma/EgammaTools/interface/GainSwitchTools.h"
+
+ConversionGSCrysFixer::ConversionGSCrysFixer(const edm::ParameterSet& config)
+{
+  // singleLegConversions = EGRefinedSCFixer
+  getToken(inputConvsToken_, config, "conversions");
+  //  getToken(inputSingleLegConvsToken_, config, "scMaps");
+  //  getToken(refinedSCsToken_, config, "scMaps");
+  //  getToken(refinedSCMapToken_, config, "scMaps");
```

Comment:
is this commented out code needed?
Please remove or add comments inline in the code why the commented out block is relevant